android/quest/src/main/res/values/strings.xml

```diff
@@ -123,7 +123,7 @@
     <string name="submit_questionnaire">Submit</string>
     <string name="invalid_base_resource">The base resource for GeoWidgetConfiguration MUST be Location</string>
     <string name="no_search_coonfigs_provided">No configs provided for search bar</string>
-    <string name="no_found_locations_matching_text">No location found matching text \"%1$s\"</string>
+    <string name="no_found_locations_matching_text">No caregiver found matching text \"%1$s\"</string>
```

Comment:
@owais-vd This change should not be here. But pushed to the Strings XML file of the build variant

Reply:
@f-odhiambo done